config/telegraf-spdk.conf

```diff
@@ -17,6 +17,9 @@
   tag_keys = ["name"]
   json_query = "result.bdevs"
 
+[[inputs.nginx]]
```

Comment:
I'm not sure why nginx stats are reported via a conf file with spdk in the name.  Are they related in some way?

Reply:
I can rename the file and remove spdk from the name, thanks @seroyer good comment